<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>memberCatalogueV2</class>
 <widget class="QDialog" name="memberCatalogueV2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>1014</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	background: #FFE5D0;
	font-family: Montserrat;
}</string>
  </property>
  <widget class="QLabel" name="logoImage">
   <property name="geometry">
    <rect>
     <x>404</x>
     <y>70</y>
     <width>441</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/resources/images/bblLogo.png</pixmap>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>1181</width>
     <height>761</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{
	font-weight: bold;
	font-size: 18px;
	color: rgb(255, 136, 108);
	font: 500 18px &quot;Montserrat&quot;;

	background-color: rgb(255, 229, 208);

}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>0</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Sort By</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Title</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Author</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Genre</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Release date</string>
     </property>
    </item>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>1181</width>
      <height>711</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QScrollArea{
	background-color: rgb(255, 229, 208);
}

/* vertical scrollbar*/
QScrollBar:vertical{
	border: none;
	background-color: rgb(255, 229, 208);
	margin: 15px 0 15px 0;
	width: 14px;
}

/* hande bar verticall */
QScrollBar::handle:vertical{
	border-radius: 5px;
	background-color: rgb(255, 213, 164);
	min-height: 30px;
}
QScrollBar::handle:vertical:hover{
	background-color: rgb(255, 210, 158);
}

/* top button scrollbar */
QScrollBar::sub-line:vertical{
	border: none;
	background-color: rgb(255, 180, 126);
	height: 15px;
	border-top-left-radius: 7px;
	border-top-right-radius: 7px;
	subcontrol-position: top;
	subcontrol-origin: margin;
}
QScrollBar::sub-line:vertical:hover{
	background-color: rgb(255, 85, 127);
}
QScrollBar::sub-line:vertical:pressed{
	background-color: rgb(255, 0, 127);
}

/* bottom button scrollbar */
QScrollBar::add-line:vertical{
	border: none;
	background-color: rgb(255, 180, 126);
	height: 15px;
	border-bottom-left-radius: 7px;
	border-bottom-right-radius: 7px;
	subcontrol-position: bottom;
	subcontrol-origin: margin;
}
QScrollBar::add-line:vertical:hover{
	background-color: rgb(255, 85, 127);
}
QScrollBar::add-line:vertical:pressed{
	background-color: rgb(255, 0, 127);
}</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1165</width>
       <height>918</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QFrame" name="frame">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>900</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QScrollArea{
	background-color: rgb(255, 229, 208);
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QWidget" name="horizontalLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>1141</width>
           <height>171</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="book_1">
          <item>
           <widget class="QLabel" name="img_2">
            <property name="text">
             <string>Book 1</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="label_17">
              <property name="styleSheet">
               <string notr="true">QLabel {
	font-weight:bold;
	font-size: 14px;

	
}</string>
              </property>
              <property name="text">
               <string>50 Shades of Swag</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>The FitnessGramâ„¢ Pacer Test is a multistage aerobic capacity test that progressively 
gets more difficult as it continues. 
The second time you fail to complete a lap before the sound, 
your test is over. The test will begin on the word start. On your mark, get ready, start.
</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>Status:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_9">
                <property name="styleSheet">
                 <string notr="true">QLabel{
	color: rgb(85, 170, 0);
}</string>
                </property>
                <property name="text">
                 <string>Available</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_2">
              <property name="styleSheet">
               <string notr="true">QPushButton{
	background-color:rgb(85, 170, 0);
	font-weight: bold;
	color: rgb(0, 85, 0);
}</string>
              </property>
              <property name="text">
               <string>READ</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_3">
              <property name="styleSheet">
               <string notr="true">QPushButton{
	background-color: rgb(255, 0, 0);
	color: rgb(170, 0, 0);
	font-weight: bold;
}</string>
              </property>
              <property name="text">
               <string>RESERVE</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="horizontalLayoutWidget_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>180</y>
           <width>1141</width>
           <height>171</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="book_2">
          <item>
           <widget class="QLabel" name="img_3">
            <property name="text">
             <string>Book 2</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="label_18">
              <property name="styleSheet">
               <string notr="true">QLabel {
	font-weight:bold;
	font-size: 14px;

	
}</string>
              </property>
              <property name="text">
               <string>50 Shades of Swag</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>The FitnessGramâ„¢ Pacer Test is a multistage aerobic capacity test that progressively 
gets more difficult as it continues. 
The second time you fail to complete a lap before the sound, 
your test is over. The test will begin on the word start. On your mark, get ready, start.
</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>Status:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_12">
                <property name="styleSheet">
                 <string notr="true">QLabel{
	color: rgb(85, 170, 0);
}</string>
                </property>
                <property name="text">
                 <string>Available</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_4">
              <property name="styleSheet">
               <string notr="true">QPushButton{
	background-color:rgb(85, 170, 0);
	font-weight: bold;
	color: rgb(0, 85, 0);
}</string>
              </property>
              <property name="text">
               <string>READ</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_5">
              <property name="styleSheet">
               <string notr="true">QPushButton{
	background-color: rgb(255, 0, 0);
	color: rgb(170, 0, 0);
	font-weight: bold;
}</string>
              </property>
              <property name="text">
               <string>RESERVE</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="horizontalLayoutWidget_4">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>360</y>
           <width>1141</width>
           <height>171</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="book_3">
          <item>
           <widget class="QLabel" name="img_4">
            <property name="text">
             <string>Book 3</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QLabel" name="label_19">
              <property name="styleSheet">
               <string notr="true">QLabel {
	font-weight:bold;
	font-size: 14px;

	
}</string>
              </property>
              <property name="text">
               <string>50 Shades of Swag</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>The FitnessGramâ„¢ Pacer Test is a multistage aerobic capacity test that progressively 
gets more difficult as it continues. 
The second time you fail to complete a lap before the sound, 
your test is over. The test will begin on the word start. On your mark, get ready, start.
</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_14">
                <property name="text">
                 <string>Status:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_15">
                <property name="styleSheet">
                 <string notr="true">QLabel{
	color: rgb(85, 170, 0);
}</string>
                </property>
                <property name="text">
                 <string>Available</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_6">
              <property name="styleSheet">
               <string notr="true">QPushButton{
	background-color:rgb(85, 170, 0);
	font-weight: bold;
	color: rgb(0, 85, 0);
}</string>
              </property>
              <property name="text">
               <string>READ</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_7">
              <property name="styleSheet">
               <string notr="true">QPushButton{
	background-color: rgb(255, 0, 0);
	color: rgb(170, 0, 0);
	font-weight: bold;
}</string>
              </property>
              <property name="text">
               <string>RESERVE</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="horizontalLayoutWidget_5">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>540</y>
           <width>1141</width>
           <height>171</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="book_4">
          <item>
           <widget class="QLabel" name="img_5">
            <property name="text">
             <string>Book 4</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QLabel" name="label_20">
              <property name="styleSheet">
               <string notr="true">QLabel {
	font-weight:bold;
	font-size: 14px;

	
}</string>
              </property>
              <property name="text">
               <string>50 Shades of Swag</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_16">
              <property name="text">
               <string>The FitnessGramâ„¢ Pacer Test is a multistage aerobic capacity test that progressively 
gets more difficult as it continues. 
The second time you fail to complete a lap before the sound, 
your test is over. The test will begin on the word start. On your mark, get ready, start.
</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="label_21">
                <property name="text">
                 <string>Status:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_22">
                <property name="styleSheet">
                 <string notr="true">QLabel{
	color: rgb(85, 170, 0);
}</string>
                </property>
                <property name="text">
                 <string>Available</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_7">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <spacer name="horizontalSpacer_8">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_8">
              <property name="styleSheet">
               <string notr="true">QPushButton{
	background-color:rgb(85, 170, 0);
	font-weight: bold;
	color: rgb(0, 85, 0);
}</string>
              </property>
              <property name="text">
               <string>READ</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_9">
              <property name="styleSheet">
               <string notr="true">QPushButton{
	background-color: rgb(255, 0, 0);
	color: rgb(170, 0, 0);
	font-weight: bold;
}</string>
              </property>
              <property name="text">
               <string>RESERVE</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="viewCatalogue">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>190</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>-1</pointsize>
     <weight>62</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: rgba(255, 229, 208,0);
	color: rgb(255, 136, 108);
	font: 500 14px &quot;Montserrat&quot;;
}</string>
   </property>
   <property name="text">
    <string>CATALOGUE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="viewAccount">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>190</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: rgba(255, 229, 208,0);
	color: rgb(255, 136, 108);
	font: 500 14px &quot;Montserrat&quot;;
}</string>
   </property>
   <property name="text">
    <string>ACCOUNT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="viewCatalogue_2">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>190</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>-1</pointsize>
     <weight>62</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: rgba(255, 229, 208,0);
	color: rgb(255, 136, 108);
	font: 500 14px &quot;Montserrat&quot;;
}</string>
   </property>
   <property name="text">
    <string>LOG OUT</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
